draft-ietf-rats-architecture.md

```diff
@@ -316,6 +316,26 @@ Relying Party:
 : A remote server that will securely grant the Attester
 permission to continue operating (i.e., not reboot) for a period of time
 
+## FIDO Biometric Authentication
+
+In the FIDO protocol, it is the device in the user’s hand that authenticates the human user by biometrics like fingerprints or PIN and password.
+The comparison of the input from the user to the reference or expected values is done by the device, often a mobile phone, not the relying party.
+For the relying party to know that the authentication is trustworthy the relying party needs to know that the authenticator part of the device is trustworthy.
+The FIDO protocol employs attestation for this. 
```

Comment:
```suggestion
The FIDO protocol employs evidence generation for this. 
```

Reply:
I know... that is already standard lingo, but I really think that Attestation is too confusing and does not add meaning here.

Reply:
The FIDO protocol does employ attestation. The FIDO protocol defines an attestation protocol of its own. It also can use other attestation protocols.

Conceptually, a FIDO authenticator first runs an attestation protocol to prove itself trustworthy to the relying party (no separation of Verifier and RP in FIDO). Second, it authenticates the user and tells the RP the result. When it is telling the RP the result, it is not running an attestation protocol.

Reply:
Laurence's explanation makes sense to me, I am fine with "remote attestation" in this sentence (but add "remote").